lib/common.js

```diff
@@ -1,35 +1,26 @@
 'use strict';
 
-const common = {};
-module.exports = common;
-
 // Forward an event from one EventEmitter to another.
 //   from - EventEmitter to listen for event
 //   to - EventEmitter to emit event on
 //   eventName - name of the event
 //   newEventName - name of the forwarded event (optional)
 //
-common.forwardEvent = (from, to, eventName, newEventName = eventName) => {
-  from.on(eventName, (...eventArgs) => {
-    to.emit(newEventName, ...eventArgs);
-  });
-};
+const forwardEvent = (from, to, eventName, newEventName = eventName) =>
+  from.on(eventName, (...eventArgs) => to.emit(newEventName, ...eventArgs));
```

Comment:
As far as I remember, we decided to use full arrow function syntax (with braces) when we don't need the return value, as opposed to using an expression style lambdas.